docs/index.rst

```diff
@@ -30,51 +30,64 @@ Virtualenv
+``virtualenv`` is a tool to create isolated Python environments with its own copy of
+a working Python environment that is isolated from the rest of the system.
 
-- is slower (by not having the ``app-data`` seed method),
-- is not as extendable,
-- cannot create virtual environments for arbitrarily installed python versions (and automatically discover these),
-- is not upgrade-able via `pip <https://pip.pypa.io/en/stable/installing/>`_,
-- does not have as rich programmatic API (describe virtual environments without creating them).
+.. note::
+  Since Python ``3.3``, a subset of it has been integrated into the standard library under the `venv module <https://docs.Python.org/3/library/venv.html>`_. The
+  ``venv`` module does not offer all features of this library. See the :doc:`differences <differences_with_venv>` betwwen virtualenv and venv.
 
-The basic problem being addressed is one of dependencies and versions, and indirectly permissions.
-Imagine you have an application that needs version ``1`` of ``LibFoo``, but another application requires version
-``2``. How can you use both these libraries? If you install everything into your host python (e.g. ``python3.8``)
-it's easy to end up in a situation where two packages have conflicting requirements.
+The need for isolation of Python environments comes when there are two Python
+libraries with their own set of dependencies, Python versions and permissions.
+Python applications will often use external libraries that are not part of the standard Python
+library. The applications will often require a specific version  of that library where a particular
+bug has been fixed. Imagine you have an application that needs version ``1.2.3`` of ``foo``,
+but another application requires version ``2.3.4``. Installing one version of the library would
+make the other application unable to run. So, If you install everything into your system Python
+it's easy to end up in a situation where two applications have conflicting requirements.
 
 Or more generally, what if you want to install an application *and leave it be*? If an application works, any change
 in its libraries or the versions of those libraries can break the application. Also, what if you can't install packages
-into the global ``site-packages`` directory, due to not having permissions to change the host python environment?
+into the global ``site-packages`` directory, due to not having permissions to change the system Python environment?
 
 In all these cases, ``virtualenv`` can help you. It creates an environment that has its own installation directories,
 that doesn't share libraries with other virtualenv environments (and optionally doesn't access the globally installed
 libraries either).
 
-Useful links
-------------
+Quick Start
+-----------
+The recommended way to install virtualenv is through pip: (for other methods see :doc:`Installation <installation>`)
```

Comment:
I strongly disagree with this. installing virtualenv with pip causes all kind of issues I'm not comfortable recommending.

Reply:
I will replace with `pipx` then, thanks (y)

Reply:
I don't think needs replaced, needs to be removed. Maybe a link to the install section but no need to point out any method in particular. The user might even pull it in from its operating system package manager, it should not matter.

Reply:
I wanted to write a "Quick Start" and a "Tutorial" mostly aimed at beginners, who would come to the docs. Since the installation and activation part is best delegated to its original documentation (which is a little verbose IMO), do you want me to remove this section entirely?

Reply:
I'd say yes.

Reply:
Sure, then what would you suggest would be the best way to have a guide/tutorial that is well suited for beginners, and doesn't have to go through pages of documentation to figure out how to use `virtualenv`

Reply:
I have removed the Quick Start section